README.md

```diff
@@ -0,0 +1,104 @@
+# The Krun Linux Kernel
+
+This repo houses the customised Linux-4.9.0 Kernel for use with the
+[Krun Benchmarking System](https://github.com/softdevteam/krun).
+
+This kernel has additional system calls for low-latency access to several MSRs
+which Krun uses for benchmarking. Although the existing `msr' (and `rmsr')
```

Comment:
I think you mean `msr` rather than `msr' (same for rmsr), because Markdown...

Reply:
yep